plugins/inspect/services_inspector.rb

```diff
@@ -42,6 +47,13 @@ def has_systemd(system)
     false
   end
 
+  def has_redhat_chkconfig(system)
+    system.run_command("chkconfig", "--version")
+    true
+  rescue Cheetah::ExecutionFailed
```

Comment:
Checking for the existence of a tool, where it not being existant is a valid case, should not be done through exceptions. I would rather have a method in `System` which encapsulates that (something like `has_command?`) and return a boolean as result.